src/sentry/models/docintegrationavatar.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey
+
+from . import AvatarBase
+
+
+class DocIntegrationAvatar(AvatarBase):
+    """
+    A DocIntegrationAvatar associates a DocIntegration with a logo photo File.
+    """
+
+    FILE_TYPE = "avatar.file"
+
+    doc_integration = FlexibleForeignKey("sentry.DocIntegration", related_name="avatar")
+    avatar_type = models.PositiveSmallIntegerField(default=0, choices=((0, "upload"),))
+    date_added = models.DateTimeField(default=timezone.now, null=True)
```

Comment:
This doesn't need to be nullable, is only like that on `DefaultFieldsModel` so that it can work with existing models as well as new ones.